### ISSUE TITLE
Error trying to re-set line length

### ISSUE THREAD

**Issue by EricPacefactory** [spyder-ide/spyder#16302]
### What steps will reproduce the problem?

1. Click on top-right menu button on the editor pane and open it as a new window
2. In new editor window, go to: Tools > Preferences
3. In preferences go to: Completion and linting > Code style and formatting
4. For the 'Maximum allowed line length' setting, click the down arrow once, then the up arrow once then click apply. This warning comes up.

Notes: I have the line length set to 120 by default, but the editor always defaults to 80(-ish?) on startup, so this sequence is just used to try and set it back to 120. This error doesn't seem to occur if I don't first open the editor in a new window

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/home/eo/.virtualenvs/spyder/lib/python3.8/site-packages/spyder/plugins/editor/plugin.py", line 407, in _rpc_call
    meth(*args, **kwargs)
  File "/home/eo/.virtualenvs/spyder/lib/python3.8/site-packages/spyder/plugins/editor/plugin.py", line 1599, in call_all_editorstacks
    method = getattr(editorstack, method)
TypeError: getattr(): attribute name must be string
```

## Versions

* Spyder version: 5.1.1 
* Python version: 3.8.10
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Linux 5.4.0-80-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.26.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.0 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.1.0 (OK)
nbconvert >=4.0               :  6.1.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.2 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0                :  2.10.2 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.1;<1.3.0          :  1.2.1 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.1.1 (OK)
qtpy >=1.5.0                  :  1.10.0 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  57.4.0 (OK)
sphinx >=0.6.6                :  4.1.2 (OK)
spyder_kernels >=2.1.0;<2.2.0 :  2.1.0 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.5 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.2.1 (OK)

# Optional:
cython >=0.21                 :  None (OK)
matplotlib >=2.0.0            :  None (OK)
numpy >=1.7                   :  None (OK)
pandas >=1.1.1                :  None (OK)
scipy >=0.17.0                :  None (OK)
sympy >=0.7.3                 :  None (OK)
```

**Comment by ccordoba12** [comment]
Hey @EricPacefactory, thanks for reporting. We'll try to fix this bug in our next version (5.2.0), to be released in mid October.

**Comment by mrclary** [comment]
#16439 is a duplicate of this issue, however it has more info in the conversation, so closing this one.
